recipes-debian/libpam-ssh/libpam-ssh_debian.bb

```diff
@@ -0,0 +1,43 @@
+SUMMARY = "This pluggable authentication module (PAM) provides single sign-on using secure shell (SSH) keys."
+LICENSE = "BSD-3-Clause | GPLv3+"
```

Comment:
Why did you think so?

https://metadata.ftp-master.debian.org/changelogs//main/libp/libpam-ssh/libpam-ssh_2.3+ds-1_copyright

Reply:
I based on the following statements.
```
Files: *
Copyright:
 2006-2019 Wolfgang Rosenauer <wolfgang@rosenauer.org>
 1999, 2000, 2001, 2002, 2004 Andrew J. Korty <ajk@iu.edu>
License: BSD-3-clause
```

```
Files: debian/*
Copyright:
 2012-2019 Jerome Benoit <calculus@rezozer.net>
 2008-2010 Jens Peter Secher <jps@debian.org>
 2004-2005 Aurelien Labrosse <aurelien.labrosse@free.fr>
License: GPL-3+
```

I am sorry, how should I write correctly?

Reply:
I'd like to know the reason why you choose only BSD-3-clause and GPL-3+ from the following licenses. (Other licenses are not related to the built artifacts?)

```
$ wget -O - https://metadata.ftp-master.debian.org/changelogs//main/libp/libpam-ssh/libpam-ssh_2.3+ds-1_copyright | grep ^License: | sort | uniq
...
License: AdHoc-0
License: BSD-2-clause
License: BSD-3-clause
License: BSD-4-clause-Niels-Provos
License: GPL-3+
License: ISC
License: ISC and ISC-IBM
License: ISC-IBM
License: public-domain-0
License: public-domain-1
```

In addition, `"BSD-3-Clause | GPLv3+"` means that the package is licensed under BSD-3-clause "OR" GPL-3+.
If you'd like to express "AND", you should use `&` instead of `|`.

Reply:
> I'd like to know the reason why you choose only BSD-3-clause and GPL-3+ from the following licenses.
> (Other licenses are not related to the built artifacts?)

From the description of 'Files: *', I thought that the all pam_ssh part was BSD-3-Clause.
However, it was wrong.
To include other licenses, I added them to `LICENSE` variable with the standard keyword.

When I added AdHoc-0 to `LICENSE` variable, it printed WARNING.
This is because AdHoc-0 is not standard keywords.
To correct this, I added AdHoc-0 to `NO_GENERIC_LICENSE` array.


> In addition, "BSD-3-Clause | GPLv3+" means that the package is licensed under BSD-3-clause "OR" GPL-3+.
> If you'd like to express "AND", you should use & instead of |.

Thank you. I fixed it.

Reply:
@AokiNobuhiko-Metatechno Thanks, LGTM.

@masami256 If it's okay with you, please merge this PR.